frontend/src/Components/CustomizedInputBase.js

```diff
@@ -25,20 +27,29 @@ export function CustomizedInputBase(props) {
   const [input, setInput] = React.useState('');
   const [subgraph, setSubgraph] = React.useState('');
   const Search = async (event) => {
+    let res;
     event.preventDefault();
     // console.log(input)
     props.apiCalled(true)
-
-    // const res = await axios.post('http://127.0.0.1:5000/api/v1/dashboard', {
-    //   input: input,
-    //   subgraph: subgraph,
-    // })
+    
+    try{
+      res = await axios.post(`${backend_url}/api/v1/dashboard`, {
+      input: input,
+      subgraph: subgraph,
+    })
+    props.setData(res.data?.output);
+    props.apiCalled(false)
+  }
+    catch{
+      props.setData("");
+      props.setData([{"decimals":9,"id":"0xcf0c122c6b73ff809c693db761e7baebe62b6a2e","name":"FLOKI","symbol":"FLOKI","transferCount":274254},{"decimals":18,"id":"0x320623b8e4ff03373931769a31fc52a4e78b5d70","name":"Reserve Rights","symbol":"RSR","transferCount":"121409"},{"decimals":18,"id":"0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc","name":"Hop","symbol":"HOP","transferCount":"78497"},{"decimals":18,"id":"0xa2cd3d43c775978a96bdbf12d733d5a1ed94fb18","name":"Chain","symbol":"XCN","transferCount":"70327"},{"decimals":9,"id":"0xa67e9f021b9d208f7e3365b2a155e3c55b27de71","name":"KleeKai","symbol":"KLEE","transferCount":"37061"}]);
```

Comment:
I think we want this to display a "The query generation failed" or something to that effect vs displaying a random dataset

Reply:
yep. I will change it. Just to see everything is working fine I have set that.